README.md

```diff
@@ -37,6 +39,8 @@ $ wstool set -t src isri-aist/QpSolverCollection git@github.com:isri-aist/QpSolv
 $ wstool update -t src
 ```
 
+> `wstool` can be installed with `pip install wstool` 
```

Comment:
`wstool` can be installed with `apt install python3-wstool` or `pip install wstool`.